app/src/main/res/values-es-rES/strings.xml

```diff
@@ -1764,4 +1764,20 @@ the Select` button. When clicked it closes all the previously selected tabs -->
     <string name="deprecated_version_dialog_store_button">Instalar…</string>
     <string name="security_options_permission_fine_location_warning">Wolvic sólo puede obtener tu ubicación aproximada. Esto se puede cambiar en la configuración del sistema.</string>
     <string name="hamburger_menu_toggle_passthrough">Passthrough</string>
+    <string name="user_feedback_tooltip">Compartir ideas y opiniones</string>
+    <string name="rCN_first_launch_clickable_privacy_text">Política de privacidad</string>
+    <string name="rCN_first_launch_agree">Aceptar</string>
+    <string name="rCN_first_launch_disagree">No estoy de acuerdo</string>
+    <string name="rCN_first_launch_confirmation_body">\\Si seleccionas \"No estoy de acuerdo\", lamentablemente ya no podremos brindarle los servicios.
+\n
+\nPuede leer el Acuerdo de usuario y la Política de privacidad completos.</string>
+    <string name="rCN_first_launch_confirmation_agree">Aceptar y continuar</string>
+    <string name="rCN_first_launch_title">Acuerdo del usuario y la notificación de la política de privacidad</string>
+    <string name="rCN_first_launch_clickable_terms_text">Acuerdo del Usuario</string>
+    <string name="rCN_first_launch_body">¡Bienvenidos a “Wolvic”!
+\n
+\nRegistrarte y usar \"Wolvic\" requiere que aceptes el Acuerdo del usuario y la Política de privacidad, o de lo contrario no podemos brindarle los servicios. Lee detenidamente la información y haz clic en \"Aceptar\" para continuar. Si hazes clic en \"No estoy de acuerdo\", no podrás continuar usando \"Wolvic\".
+\n
+\nPuedes leer el Acuerdo de usuario y la Política de privacidad completos.</string>
```

Comment:
In this sentences it's used the term "Acuerdo de usuario". I think we must be consistent on the use of this term.